src/ukify/ukify.py

```diff
@@ -1325,10 +1325,10 @@ def apply_config(self, namespace, section, group, key, value) -> None:
         else:
             conv = lambda s:s
 
```

Comment:
I think it makes sense to squash this commit into the prior commit that introduces multi-dtb support to ukify in the first place